context/tasks/runmode-pm-audit.md

```
@@ -0,0 +1,109 @@
+   - `_llm_step(...)`
+5. `penguin/project/`
+   - `task_executor.py`
+   - `workflow_orchestrator.py`
+   - `manager.py`
+   - `validation_manager.py`
+6. `penguin/web/routes.py`
+   - task execution / clarification resume / serialized task payloads
+
+## Verified Findings
+
+### What looks solid enough right now
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)